www/js/app.js

```diff
@@ -899,17 +887,20 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
      * @param {DirEntry} dirEntry The directory entry of the article to read
      */
     function readArticle(dirEntry) {
+        // If we are not loading the landing page, remove focus from the UI elements (for both jQuery and SW modes)
```

Comment:
It would be worth explaining a bit more why we have to do that

Reply:
Done.